test/platform.php.test.php

```diff
@@ -2,16 +2,14 @@
 
 // Run tests from the repository root directory
 // Linux:
-// $ composer install && ./vendor/bin/phpunit test/MailCheckerTest.php
+// $ composer install && vendor/bin/phpunit test/platform.php.test.php
 // Windows:
-// > composer install && .\vendor\bin\phpunit.bat test\MailCheckerTest.php
+// > composer install && .\vendor\bin\phpunit.bat test\platform.php.test.php
 
-require_once __DIR__ . '/../platform/php/MailChecker.php';
```

Comment:
We use the autoloader now, to ensure the composer setup is correct

Reply:
sadly `npm run test:php` does not run with these changes, I'm bringing it back:

```
require_once __DIR__ . '/../platform/php/MailChecker.php';
```

Reply:
indeed, we don't rely on composer currently (see package.json for the cross-language support)